python/ray/air/tests/mocked_wandb_integration.py

```diff
@@ -32,14 +30,11 @@ def __str__(self):
 
 
 class _FakeConfig:
```

Comment:
Will this be written by multiple `_MockWandbAPI`s? Will there be any contention there if the class is not multi-process safe?

Reply:
None of the tests actually do multiple trials writing to the API right now. Maybe we can make this a named actor with a queue inside.

Reply:
yeah, but I think since this is a generic mock class that may potentially be used for testing tune wandb integration, we should either fix that or add a todo and warn people about it.

Reply:
Got it, I will fix this then.